contrib/mesos/pkg/node/node.go

```diff
@@ -55,6 +55,17 @@ func Create(
 		},
 		Status: api.NodeStatus{
 			Phase: api.NodePending,
+			// WORKAROUND(sttts): make sure that the Ready condition is the
```

Comment:
workaround .. is there a more desirable solution? if so, should we file an issue to track it?